solr/core/src/java/org/apache/solr/response/SolrQueryResponse.java

```diff
@@ -139,30 +142,63 @@ public ReturnFields getReturnFields() {
 
   /**
    * If {@link #getResponseHeader()} is available, set {@link #RESPONSE_HEADER_PARTIAL_RESULTS_KEY}
-   * flag to true.
+   * attribute to true or "omitted" as required.
    */
-  public void setPartialResults() {
+  public void setPartialResults(SolrQueryRequest req) {
```

Comment:
Should this be `static` now?

Reply:
There's not really a point to making it static, the object already exists in every location it's used.